Backend/dev/test-tool/local-api/server.py

```
@@ -933,6 +939,84 @@ def _is_localhost(host: str) -> bool:
+
+
+def _register_dev_user(ssh_user: str, host: str, port: int, identity: str | None, dev_name: str) -> dict:
+    """Ensure dev_name is in the remote registry; return their entry."""
+    registry = _remote_read_registry(ssh_user, host, port, identity)
+    users = registry.get("users", {})
+
+    if dev_name in users:
+        return users[dev_name]
+
+    entry = {
+        "dir": f"/tmp/{dev_name}/nammayatri",
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)